fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/BlockBreakEffectsCallback.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.event;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.Entity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+/**
+ * Callback for the effects displayed when a block is broken (particles and sounds)
+ *
+ * <p>Upon return:
```

Comment:
Description of return values should be on the method for the javadoc.

Reply:
OK, I was just coping the other events in this package.

Reply:
Every other event in ``fabric-events-interaction-v0`` has the return values described in the JavaDoc for the class.

Reply:
We should change those. We should move return values to where they belong. I think some people just carried bad habits from some other project over.

Reply:
I moved them.